docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,27 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
+### Scenario Installer
+
+### OSTree Content
+
+### Lazy Sync
+
+### Docker V2
+
+### Mirror On Sync
+
+### Remote Execution Integration
+
+### Capsule Status on Proxy
```

Comment:
Proxy detail page was extended to provide statistics about synchronized content grouped by lifecycle environments, usage of pulp storage to warn users when the capsule is running out of disc space and information about synchronization status. The latter displays progress of current tasks when the synchronization is running. It also allows for starting and cancelling the synchronization process.

The ability to cancel synchronizations as well as showing the content counts are now reflected in the cli. You can find it in commands `hammer capsule content cancel-synchronization` and `hammer capsule content info`. Please note that the command `capsule` has became an alias to `proxy` and they share the same subcommands.